homeassistant/components/switch/xiaomi_miio.py

```diff
@@ -207,6 +291,26 @@ def async_update(self):
             self._state = None
             _LOGGER.error("Got exception while fetching the state: %s", ex)
 
+    # pylint: disable=no-self-use
+    async def async_set_power_mode(self, mode: str):
+        """Set the power mode."""
+        return
+
+    # pylint: disable=no-self-use
+    async def async_set_wifi_led_on(self):
+        """Turn the wifi led on."""
+        return
+
+    # pylint: disable=no-self-use
+    async def async_set_wifi_led_off(self):
+        """Turn the wifi led on."""
+        return
+
+    # pylint: disable=no-self-use
+    async def async_set_power_price(self, price: int):
+        """Set the power price."""
+        return
```

Comment:
Should these calls raise an exception instead of failing silently, if they are somehow invoked?

Reply:
I checked other components: If a feature/service isn't supported the method just returns.